jolt-core/src/subprotocols/sumcheck.rs

```diff
@@ -100,11 +100,11 @@ impl<F: JoltField> CubicSumcheckParams<F> {
         if *l == F::one() && *r == F::one() {
             *eq
         } else if *l == F::one() {
-            *r * eq
+            *r * *eq
```

Comment:
we should make it so that you can perform arithmetic operations on `&F` values without dereferencing
https://stackoverflow.com/a/59547146

Reply:
Binius' fields don't implement these by reference so it's hard for us to do it with the same automatic method. #380